oc-chef-pedant/lib/pedant/rspec/search_util.rb

```diff
@@ -230,12 +231,20 @@ def setup_multiple_objects(object_type, number_to_create=5)
             objects.each do |o|
               send(object_add_method_symbol, admin_requestor, o)
             end
+            if objects.length > 0
+              Pedant::Utility.wait_until_queues_are_empty(i = 10, safety_sleep: false, quiet: true)
+            end
           end
 
           after :each do
             objects.each do |o|
               send(object_delete_method_symbol, admin_requestor, o['name'])
             end
+            # Technically we should call this here as well, but if the
```

Comment:
would the names of the objects overlap in anyway? If so, is the queue drained and messages evaluated to completion in order?